### PR TITLE
Mappings reverts

### PR DESCRIPTION
**Pull request by stefan-tudor**
- Reverts `response.result` to old mappings
- Reverts `threeDSTwoMaxTimeout` type

### CODE REVIEW

File: types/JudoTypes.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export interface JudoConfiguration {
     mobileNumber?: string,
     phoneCountryCode?: string,
     emailAddress?: string,
-    threeDSTwoMaxTimeout?: number,
+    threeDSTwoMaxTimeout?: string,
```
Comment (zfoltin) [inline types/JudoTypes.tsx:24]: Why do we have to revert this change?

Comment (stefan-tudor) [reply types/JudoTypes.tsx:24]: to only have the changes from `ios/Classes/Wrappers/RNWrappers.m` in the next release
